certified-connectors/Nodefusion Portal/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "6f2a7cda-d35a-40ac-b913-d3ec4066c42a",

```

Comment:
@hkusulja As per the Microsoft security policies, it is not allowed to post any security and PII information in any connector artifacts. Please remove the clientId value from your swagger. You can replace it with `"[[Dummy]]"` or `"Your Client Id"`.

Reply:
ok, this will be specified in a portal